### PR TITLE
19w13a mappings

### PR DESCRIPTION
**Pull request by Runemoro**

### CODE REVIEW

File: mappings/net/minecraft/util/math/Vec3i.mapping [hunk 1]
```diff
@@ -3,9 +3,26 @@ CLASS fq net/minecraft/util/math/Vec3i
...
+		ARG 2 treatAsBlockPos
+	METHOD a isWithinDistance (Lfq;D)Z
+		ARG 1 vec
+		ARG 2 distance
 	METHOD d crossProduct (Lfq;)Lfq;
-	METHOD m squaredDistanceTo (Lfq;)D
+		ARG 1 vec
+	METHOD m getSquaredDistance (Lfq;)D
+		ARG 1 vec
+	METHOD n getTaxicabDistance (Lfq;)I
```
Comment (liach) [inline mappings/net/minecraft/util/math/Vec3i.mapping]: Is taxicab or manhattan distance better known?

File: mappings/net/minecraft/util/math/Vec3i.mapping [hunk 2]
```diff
@@ -3,9 +3,26 @@ CLASS fq net/minecraft/util/math/Vec3i
 	FIELD b y I
 	FIELD c z I
 	FIELD e ZERO Lfq;
-	METHOD a squaredDistanceToCenter (DDDZ)D
+	METHOD a getDistanceSq (DDDZ)D
```
Comment (liach) [inline mappings/net/minecraft/util/math/Vec3i.mapping]: `getSquaredDistance`

Comment (Runemoro) [reply mappings/net/minecraft/util/math/Vec3i.mapping]: Fixed

**Comment by liach** [comment]
Oh, you need to resolve merge conflicts as well.

File: mappings/net/minecraft/entity/ai/brain/WalkTarget.mapping [hunk 3]
```diff
@@ -1,9 +1,9 @@
 CLASS aot net/minecraft/entity/ai/brain/WalkTarget
 	FIELD a lookTarget Lakg;
-	FIELD b speedFactor F
+	FIELD b speed F
```
Comment (Gegy) [inline mappings/net/minecraft/entity/ai/brain/WalkTarget.mapping:3]: Why this change?

File: mappings/net/minecraft/util/math/Vec3i.mapping (hunk 1 above)
Comment (Gegy) [reply mappings/net/minecraft/util/math/Vec3i.mapping]: I've only really heard Manhattan be used, imo that is more clear

File: mappings/net/minecraft/entity/ai/brain/WalkTarget.mapping (hunk 3 above)
Comment (Runemoro) [reply mappings/net/minecraft/entity/ai/brain/WalkTarget.mapping:3]: It's eventually passed to `startMovingAlong(@Nullable final Path path, final double speed)` in `EntityNavigation`, which calls it "speed", not "speedFactor".

File: mappings/net/minecraft/util/math/Vec3i.mapping (hunk 1 above)
Comment (Runemoro) [reply mappings/net/minecraft/util/math/Vec3i.mapping]: Changed it

File: mappings/net/minecraft/block/SmallTranslucentBlock.mapping [hunk 4]
```diff
@@ -0,0 +1 @@
+CLASS blb net/minecraft/block/SmallTranslucentBlock
```
Comment (Virtuoel) [inline mappings/net/minecraft/block/SmallTranslucentBlock.mapping:1]: I don't really get the meaning of the "Small" part. The methods in this class were previously part of the `GlassBlock` and `StainedGlassBlock` classes, so it would probably be more suitable as some kind of `AbstractGlassBlock`.

File: mappings/net/minecraft/block/BedrockBlock.mapping [hunk 5]
```diff
@@ -0,0 +1 @@
+CLASS blu net/minecraft/block/BedrockBlock
```
Comment (Virtuoel) [inline mappings/net/minecraft/block/BedrockBlock.mapping:1]: Despite only being used for Bedrock, it's perfectly usable as a generic spawnproof block and has nothing specific to Bedrock other than the spawnproof part. It would look rather odd if someone made use of a class called `BedrockBlock` for some breakable spawnproof block in their mod.

File: mappings/net/minecraft/block/SmallTranslucentBlock.mapping (hunk 4 above)
Comment (Runemoro) [reply mappings/net/minecraft/block/SmallTranslucentBlock.mapping:1]: I may have really misread this, and I can't check right now, but isn't it extended by G**r**assBlock?

Comment (Virtuoel) [reply mappings/net/minecraft/block/SmallTranslucentBlock.mapping:1]: Nope, definitely Glass

File: mappings/net/minecraft/block/BedrockBlock.mapping (hunk 5 above)
Comment (Runemoro) [reply mappings/net/minecraft/block/BedrockBlock.mapping:1]: Based on the obfuscated name (in alphabetical order relative to other block classes) the Mojang name starts with a `B`, so it's probably `BedrockBlock`.

It would be a really bad idea to use that in a mod since mojang may add a method at any point.

It also doesn't really makes sense to have a "spawnproof block" class, since unlike the other block classes, "spawnproof" isn't a kind of block (glass, log, etc.).

Comment (Virtuoel) [reply mappings/net/minecraft/block/BedrockBlock.mapping:1]: Hmm, good points. `BedrockBlock` does seem fine in that case.

File: mappings/net/minecraft/block/SmallTranslucentBlock.mapping (hunk 4 above)
Comment (Runemoro) [reply mappings/net/minecraft/block/SmallTranslucentBlock.mapping]: Hmm, then I'm blind. I'll change it when I get home in a few hours.

**Comment by modmuss50** [comment]
Merging due to update, if you have any more comments please open a new issue or a new PR.
